extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/ApplyHttpGetActionPortDecorator.java

```diff
@@ -90,6 +96,12 @@ public void visit(HTTPGetActionFluent<?> action) {
         } else {
             action.withNewPort(port);
         }
+
+        if (scheme == null) {
+            action.withScheme((String) null);
+        } else {
+            action.withScheme(scheme);
+        }
```

Comment:
why not simply using `action.withSchema(schema)`?

Reply:
I assume because the compiler can't determine which method to call because of the missing type

Reply:
I agree it's probably not needed by let's not block the PR for that, merging.